tests/unit_tests/test_tethys_portal/test_views/test_user.py

```diff
@@ -72,6 +97,117 @@ def test_profile(self, mock_get_user, mock_token_get_create, mock_render, mock_g
+    @mock.patch('tethys_portal.views.user.get_quota')
+    @mock.patch('tethys_portal.views.user.render')
+    @mock.patch('tethys_portal.views.user.Token.objects.get_or_create')
+    @mock.patch('tethys_portal.views.user.User.objects.get')
+    def test_profile_mfa_required_mfa_set(self, mock_get_user, mock_token_get_create, mock_render, mock_get_quota,
+                                          mock_convert_units, mock_has_mfa, _):
+        mock_request = mock.MagicMock()
+        username = 'foo'
+
+        mock_context_user = mock.MagicMock()
+        mock_get_user.return_value = mock_context_user
+
+        mock_user_token = mock.MagicMock()
+        mock_token_created = mock.MagicMock()
+        mock_token_get_create.return_value = mock_user_token, mock_token_created
+        mock_convert_units.return_value = '0 bytes'
+        mock_get_quota.return_value = {'quota': None}
+        mock_has_mfa.return_value = True
+
+        expected_context = {
+            'context_user': mock_context_user,
+            'user_token': mock_user_token.key,
+            'current_use': '0 bytes',
+            'quota': None,
+            'has_mfa': True,
+            'mfa_required': True,
+            'show_user_token_mfa': True  # Show user token b/c mfa is required and user has setup mfa
+        }
+
+        profile(mock_request, username)
+
+        mock_render.assert_called_with(mock_request, 'tethys_portal/user/profile.html', expected_context)
+
+    @override_settings(MFA_REQUIRED=False)
+    @mock.patch('tethys_quotas.utilities.log')
+    @mock.patch('tethys_portal.views.user.has_mfa')
+    @mock.patch('tethys_portal.views.user._convert_storage_units')
+    @mock.patch('tethys_portal.views.user.get_quota')
+    @mock.patch('tethys_portal.views.user.render')
+    @mock.patch('tethys_portal.views.user.Token.objects.get_or_create')
+    @mock.patch('tethys_portal.views.user.User.objects.get')
+    def test_profile_mfa_not_required_mfa_set(self, mock_get_user, mock_token_get_create, mock_render, mock_get_quota,
+                                              mock_convert_units, mock_has_mfa, _):
+        mock_request = mock.MagicMock()
+        username = 'foo'
+
+        mock_context_user = mock.MagicMock()
+        mock_get_user.return_value = mock_context_user
+
+        mock_user_token = mock.MagicMock()
+        mock_token_created = mock.MagicMock()
+        mock_token_get_create.return_value = mock_user_token, mock_token_created
+        mock_convert_units.return_value = '0 bytes'
+        mock_get_quota.return_value = {'quota': None}
+        mock_has_mfa.return_value = True
+
+        expected_context = {
+            'context_user': mock_context_user,
+            'user_token': mock_user_token.key,
+            'current_use': '0 bytes',
+            'quota': None,
+            'has_mfa': True,
+            'mfa_required': False,
+            'show_user_token_mfa': True  # Show user token b/c mfa is required and user has setup mfa
```

Comment:
Update the comment to "Show user token b/c not mfa_required" or something similar

Reply:
Done.